app/views/group_offers/index.html.slim

```diff
@@ -3,7 +3,7 @@ h1= t_title
 nav.navbar.section-navigation
   ul.list-inline
     li= button_link t_title(:new), new_group_offer_path, 'success', dimension: 'sm'
-    - if policy(GroupOfferCategory).superadmin?
+    - if policy(GroupOfferCategory).index?
```

Comment:
I am not sure if this is needed.

Reply:
I agree, it isn't as the authorize in the controller will only let superadmin and department managers access the whole action